src/server/api/routers/facilitydata.ts

```diff
@@ -33,22 +21,33 @@ export const facilityDataRouter = createTRPCRouter({
       });
     }),
 
-  getAllSector: publicProcedure.query(({ ctx }) => {
-    return ctx.prisma.sector.findMany();
+  getFacilityData: publicProcedure.query(({ ctx }) => {
+    const facilityData =  ctx.prisma.deliverables.findFirst({
+      where: { deliverableName: 'facilityData' },
+    });
+
+    console.log(facilityData);
+
+    return facilityData;
   }),
 
-  getFacilityDataFilenames: publicProcedure.query(({ ctx }) => {
-    const directoryPath = path.join(process.cwd(), "public", "facilityData");
-    // List all files in the directory
-    const files = fs.readdir(directoryPath);
-    return files;
+  getGroupData: publicProcedure.query(({ ctx }) => {
+    return ctx.prisma.deliverables.findFirst({
+      where: { deliverableName: 'groupData' },
+    });
   }),
 
```

Comment:
I acknowledge the amount of TODOs here, this will all change in the splitconfig branch soon to support the drag-n-drop functionality.

Reply:
Sounds good. I leave todo litter most places I go 😜